context/on_startup.py

```diff
@@ -30,15 +30,20 @@ def write_igv_configuration():
         assembly
     )
     for node_data in config_data["node_info"].values():
-        tracks.append(
-            {
-                "name": "{} - {}".format(
-                    node_data["node_solr_info"]["name"],
-                    node_data["file_url"]
-                ),
-                "url": node_data["file_url"]
-            }
-        )
+        track = {
+            "name": "{} - {}".format(
+                node_data["node_solr_info"]["name"],
+                node_data["file_url"]
+            ),
+            "url": node_data["file_url"]
+        }
+        if '.bam' == os.path.splitext(track['name']):
+            # assume that there is only one auxiliary file for bam igv and it's
+            # the .bai file
```

Comment:
```suggestion
            # Assume that there is only one auxiliary file for bam igv and it's the .bai file.
```
I generally prefer to have a capital letter and period, when a comment is a sentence, but your call.

Reply:
done!